plugins/harvesters/oai/OAIHarvester.inc.php

```diff
@@ -201,6 +201,8 @@ function getSets($harvesterUrl) {
 			return false;
 		}
 
+		if (!$result) return false;
```

Comment:
Sure, it's just that `false` suggests an error condition, and an empty set list appears to be valid.